.github/workflows/code_checks.yml

```diff
@@ -16,6 +21,13 @@ jobs:
         with:
           bundler-cache: true
 
+      - name: Add Lint Failure label
+        uses: actions-ecosystem/action-add-labels@v1
```

Comment:
Added lint failure before tests begin. Labels will be removed after check passes.

Reply:
What's the benefit of adding this label before linting runs and then removing it after it runs and passes? Why not wait and only add it if it fails?
The reasons I'm hesitant to add the label before linting runs are:
1. It might cause confusion for the PR author
2. It might clutter up the PR with messages about the label being added and the label being removed.

Reply:
It's more for the 'ready-to-review' .  I can remove adding the lint failures.